kroxylicious-filter-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,214 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>jar</packaging>
+
+    <name>Sample filters</name>
+    <description>Some exemplar filters</description>
+
+    <properties>
+        <kroxylicious.extension.version>@kroxy.extension.version@</kroxylicious.extension.version>
```

Comment:
funky stuff, I'm using a resource filtering to inject properties from the archetype module build using this custom delimiter. This is so that it leaves the likes of ${groupId} unreplaced, they are placeholders to be replaced by velocity during archetype generation :exploding_head: